exercises/Tidy-data-real-portal-data-SQL.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: exercise
 topic: Tidy Data
-title: Real Portal Data
+title: Improving Messy Data
```

Comment:
This will break things unless the file name (this exercise and the `solutions/`) and `exercises:` in `assignments/sql-joins.md` are updated to the new `title:`